sklearn/datasets/_twenty_newsgroups.py

```diff
@@ -391,19 +393,38 @@ def fetch_20newsgroups_vectorized(*, subset="train", remove=(), data_home=None,
 
         .. versionadded:: 0.22
 
+    as_frame : bool, default=False
+        If True, the data is a pandas DataFrame including columns with
+        appropriate dtypes (numeric, string, or categorical). The target is
+        a pandas DataFrame or Series depending on the number of
+        `target_columns`.
+
+        .. note::
+            Since the dataset is sparse, loading it as a pandas DataFrame
+            requires pandas 0.25+.
+
+        .. versionadded:: 0.24
+
     Returns
     -------
     bunch : :class:`~sklearn.utils.Bunch`
         Dictionary-like object, with the following attributes.
 
-        data: sparse matrix of shape (n_samples, n_features)
-            The data matrix to learn.
-        target: ndarray of shape (n_samples,)
-            The target labels.
+        data: {sparse matrix, dataframe} of shape (n_samples, n_features)
```

Comment:
should we say `sparse dataframe`?

Reply:
I would not use `sparse dataframe` because this is not the format used anymore. This is a pandas dataframe with some sparse columns:

```python
In [1]: from sklearn.datasets import fetch_20newsgroups_vectorized                   
In [2]: xxx = fetch_20newsgroups_vectorized(as_frame=True)
In [5]: type(xxx.data)                                                               
Out[5]: pandas.core.frame.DataFrame
In [10]: xxx.data.dtypes                                                             
Out[10]: 
00          Sparse[float64, 0.0]
000         Sparse[float64, 0.0]
0000        Sparse[float64, 0.0]
00000       Sparse[float64, 0.0]
000000      Sparse[float64, 0.0]
                    ...         
érale       Sparse[float64, 0.0]
ête         Sparse[float64, 0.0]
íålittin    Sparse[float64, 0.0]
ñaustin     Sparse[float64, 0.0]
ýé          Sparse[float64, 0.0]
Length: 130107, dtype: object
```
However, we could mention that the dataframe has sparse columns in the description.

Reply:
ok :+1: